### PR TITLE
Bugfix/sprint 7

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
my Account button Fix

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml [hunk 1]
```diff
@@ -65,6 +65,74 @@
                         <jcr:content jcr:primaryType="nt:unstructured"/>
                     </policy_165205831103087>
                 </container>
+                <text jcr:primaryType="nt:unstructured">
+                    <policy_1594215389761
+                            cq:styleDefaultClasses="cmp-form input"
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml]: @DeepaliAgrwl : pl check the policy class name here.

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@DeepaliAgrwl : pl refer to comments

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml]: updated the class name in the policy. The css for ths is present inside form text component clientlib

**Comment by DeepaliAgrwl** [comment]
@rishabh-behl  changes has been incorporated . Please check

**Review by rishabh-behl — APPROVED** [review]

**Pull request merged**